### ISSUE TITLE
Missing the 'armhfp'

### ISSUE THREAD

**Issue by luizperes** [brain-labs/brain#43]

**Comment by luizperes** [comment]
Add the 'armhfp' to the arch array

**Comment by rafaelcn** [comment]
Well, could you elaborate on this issue? Not sure I'm uderstanding

**Comment by luizperes** [comment]
Could you explain it better @onlurking?

**Comment by rafaelcn** [comment]
Is there something that we can do about this @onlurking?

**Comment by luizperes** [comment]
He fixed by including in the array of output architectures (he was trying on his mobile) on the makefile

**Comment by rafaelcn** [comment]
Should this be an issue then?

**Comment by luizperes** [comment]
I think that we should at least create a flag where the user could add their own @rafaelcn. but I see no harm in adding this to the current model

**Comment by onlurking** [comment]
Sorry, my smarthphone broke(was an Samsung S4), but I will try to reproduce this again on my new phone(which is an Samsung Galaxy J2 Prime).

This error occurred while build brain under [Termux](https://termux.com/).

**Comment by rafaelcn** [comment]
Yes, that's possible @luizperes and should be easily doable. I wasn't aware that you've tried to compile on your phone @onlurking, thanks for the feedback :wink:
